app/controllers/posts_controller.rb

```diff
@@ -2,7 +2,7 @@ class PostsController < ApplicationController
   def index
     @users = User.all
     @user = User.find_by(id: params[:user_id])
-    @posts = @user ? Post.where(author: @user) : nil
+    @posts = @user ? Post.where(author: @user).paginate(page: params[:page], per_page: 5) : nil
```

Comment:
- [ ] Kindly solve the `N+1` problem when fetching the posts and their comments as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Make%20sure%20that%20the%20N%2B1%20problem%20is%20solved%20when%20fetching%20all%20posts%20and%20their%20comments%20for%20a%20user.%20This%20is%20the%20corresponding%20wireframe).
### Hint:
```
@posts = @user.posts.includes(:comments)
```

Reply:
Hi @emmiiorji 

I don't get what you are trying to say.

Could you please try to explain more?

Reply:
Okay, I will.

We have the bullets gem installed which checks for n+1 queries. We've solved the issues it identified as n+1 queries.

Reply:
Hi @emmiiorji,

I think you are right the `bullet gem` solve the `N + 1` problem.
Ignore this comment and work on other suggestions and I will cross out this comment.

Reply:
Alright. Thanks.